romana-install/roles/romana/install-agent/tasks/files.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Ensure install path exists
+  file: path="{{ item }}" state=directory
+  with_items:
+    - "{{ romana_dir }}"
+    - "{{ romana_dir }}/bin"
```

Comment:
I think you can use the 'recurse' (or 'recursive'?) attribute to get the effect of "mkdir -p ...".